examples/bri-3/.eslintrc.js

```diff
@@ -9,7 +9,6 @@ module.exports = {
   extends: [
     'plugin:@typescript-eslint/recommended',
     'plugin:prettier/recommended',
-    'react-app',
```

Comment:
currently this is not linting frontend app so i removed it because linter was showing a warning
let's tackle adding frontend linting and including in git overflow in separate issue and PR since currently it is not that important as we have only one component in frontend repo

Reply:
i agree

Reply:
please make sure the build passes in github after this change.